sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -426,6 +428,14 @@ object RegExpExtract {
  */
 @ExpressionDescription(
   usage = "_FUNC_(str, regexp[, idx]) - Extracts a group that matches `regexp`.",
+  arguments = """
+    Arguments:
+      * str - a string expression.
+      * regexp - a string expression. a string representing a regular expression.
```

Comment:
Shall we remove the redundant `a string expression`?
```
- * regexp - a string expression. a string representing a regular expression.
+ * regexp - a string representing a regular expression.
```

Reply:
OK.